tests/src/kinAccountTest.ts

```diff
@@ -37,10 +37,10 @@ describe("KinAccount.createAccount", async () => {
 				memoText: memo
 			});
 		expect((txBuilder as any)._transactionBuilder.baseFee).toEqual(fee);
-		expect((txBuilder as any)._transactionBuilder.memo._value).toEqual(memo);
+		expect((txBuilder as any)._transactionBuilder.memo._value).toEqual('1-' + appId + '-' + memo);
```

Comment:
worth adding tests for empty and undefined memo (see comment on the potential undefined error at `TransactionBuilder`)